lib/topological_inventory/orchestrator/worker.rb

```diff
@@ -100,6 +100,13 @@ def collectors_from_sources_api
         hash
       end
 
+      def internal_url_for(path, query = nil)
+        URI.parse(api_base_url).tap do |uri|
+          uri.path = File.join("/internal/v0.0/", path)
```

Comment:
Ultimately we should do this kind of thing in a generic place in the lib directory.  There is an effort by, I think, @agrare to create a client wrapper in manageiq-api-common that can do auto-tenancy, but the general idea is you would set the version number at some common location like an initializer, so we don't have to bury it in the code itself.

Reply:
Opened https://github.com/ManageIQ/topological_inventory-orchestrator/issues/10 for this